### PR TITLE
Update about.html to link to each test's GitHub directories instead of directly linking to README.md files.

### PR DESCRIPTION
**Pull request by rniwa**

### CODE REVIEW

**Review by camillobruni — APPROVED** [review]
LGTM thanks!

**Review by camillobruni — APPROVED** [review]
LGTM thanks!

**Comment by julienw** [comment]
Can we create the branch before merging this though? :-)

File: about.html [hunk 1]
```diff
@@ -32,36 +32,45 @@ <h1 class="section-header">About Speedometer 3</h1>
...
-                                    <a href="resources/todomvc/architecture-examples/angular-complex/README.md" target="_blank">TodoMVC-Angular-Complex-DOM</a>,
-                                    <a href="resources/todomvc/architecture-examples/vue/README.md" target="_blank">TodoMVC-Vue</a>, <a href="resources/todomvc/architecture-examples/jquery/README.md" target="_blank">TodoMVC-jQuery</a>,
-                                    <a href="resources/todomvc/architecture-examples/preact-complex/README.md" target="_blank">TodoMVC-Preact-Complex-DOM</a>,
-                                    <a href="resources/todomvc/architecture-examples/svelte-complex/README.md" target="_blank">TodoMVC-Svelte-Complex-DOM</a>,
-                                    <a href="resources/todomvc/architecture-examples/lit-complex/README.md" target="_blank">TodoMVC-Lit-Complex-DOM</a>
+                                    Workloads: <a href="https://github.com/WebKit/Speedometer/tree/release/3.0/resources/todomvc/vanilla-examples/javascript-es5" target="_blank">TodoMVC-JavaScript-ES5</a>,
```
Comment (bgrins) [inline about.html:35]: GitHub question: does this mean the branch will be named "release/3.0"?

Comment (rniwa) [reply about.html:35]: Yes.

**Comment by rniwa** [comment]
> Can we create the branch before merging this though? :-)

My plan is to merge this then create `release/3.0` branch right away. That way, `release/3.0` would contain the correct URLs.

**Comment by bgrins** [comment]
> My plan is to merge this then create release/3.0 branch right away. That way, release/3.0 would contain the correct URLs.

OK, that's fine

**Comment by rniwa** [comment]
@bgrins @julienw : Could either one of you approve the change?

**Review by bgrins — APPROVED** [review]

**Comment by bgrins** [comment]
Done, missed that earlier.

**Comment by rniwa** [comment]
Thank you!

**Pull request merged**